src/js/conduit/Router.test.js

```diff
@@ -2,6 +2,124 @@ import '@babel/polyfill';
 
 import Router from './Router';
 
-test('adds 1 + 2 to equal 3', async () => {
-  expect(1 + 2).toBe(3);
+describe('Router', () => {
+  describe('export value', () => {
+    test('is function', () => {
+      expect(typeof Router).toBe('function');
+    });
+  });
+
+  describe('has everything necessary for iframe functionality', () => {
```

Comment:
@brainthinks we should review the pros and cons of property testing. I don't recall them off hand but basically some consider them pretty low value. but maybe it's okay.

Also.... could be other test tools for property testing and/or extensions to make it easier.

Reply:
check out https://github.com/pact-foundation/pact-js or other js property based testing frameworks.

Reply:
pact appears to be focused on api testing.  I ended up using jest-extended for additional object assertions